cmd/crdvalidator/README.md

```diff
@@ -0,0 +1,116 @@
+# crdvalidator
+
+## Summary
+A part of the core value proposition for RukPak is the safe upgrade of `CRD`s. As a result, this repository comes equipped with a "CRD Validator Webhook", `crdvalidator`, that will validate specified `CRD` upgrade events.
+
+
+### How to use
+
+The CRD Validator webhook will ensure that any CRD upgrade is safe as long as the incoming CRD has the `core.rukpak.io/enable-safe-upgrade-validation` annotation set to `enabled` or if it is owned by RukPak via an owner reference.
```

Comment:
nit: `owned by a rukpak provisioner` is more clear.

I guess we make an implicit assumption that other provisioners will implement the same ownerref pattern for objects that they manage. I think it's safe to make that assumption now but might be worth calling out in some kind of "provisioner suggested patterns" doc somewhere down the line.

Reply:
Yeah, we should probably create a provisioner onboarding document at some point. Probably not a huge issue now as we move closer to GA but will become important.